databases/sqlcipher/Portfile

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           clang_dependency 1.0
```

Comment:
it's a bit surprising to me that this is needed as it implies that the port is needed for building recent clangs (at least if I read the PG correctly). Given that this port wasn't present before and clangs build fine I am a bit confused about this, but must admit that I haven't spend much (or any) time on this

Reply:
This portfile is copied from sqlite3, will delete this line.